### PR TITLE
sql: Explain cop capabilities

### PR DESCRIPTION
**Pull request by morgo**
@zz-jason PTAL

### CODE REVIEW

**Comment by morgo** [comment]
@dcalvin PTAL

File: sql/understanding-the-query-execution-plan.md [hunk 1]
```diff
@@ -97,7 +97,9 @@ mysql> EXPLAIN ANALYZE SELECT count(*) FROM trips WHERE start_date BETWEEN '2017
 
 ### Introduction to task
 
-Currently, the calculation task of TiDB contains two different tasks: cop task and root task. The cop task refers to the computing task that is pushed to the KV side and executed distributedly. The root task refers to the computing task that is executed at a single point in TiDB. One of the goals of SQL optimization is to push the calculation down to the KV side as much as possible.
+Currently, the calculation task of TiDB contains two different tasks: cop task and root task. The cop task refers to the computing task that is pushed to the KV side and executed distributedly. The root task refers to the computing task that is executed at a single point in TiDB.
```
Comment (dcalvin) [inline sql/understanding-the-query-execution-plan.md]:
```suggestion
Currently, the calculation task of TiDB contains two different tasks: cop task and root task. A cop task refers to the computing task that is pushed to the KV side and executed distributedly. A root task refers to the computing task that is executed at a single point in TiDB.
```

File: sql/understanding-the-query-execution-plan.md [hunk 2]
```diff
@@ -97,7 +97,9 @@ mysql> EXPLAIN ANALYZE SELECT count(*) FROM trips WHERE start_date BETWEEN '2017
...
+
+One of the goals of SQL optimization is to push the calculation down to the KV side as much as possible. The coprocessor is able to assist in execution of SQL functions (both aggregate and scalar), SQL `LIMIT` operations, index scans and table scans. All join operations however, will be performed as root tasks.
```
Comment (dcalvin) [inline sql/understanding-the-query-execution-plan.md]:
```suggestion
One of the goals of SQL optimization is to push the calculation down to the KV side as much as possible. The coprocessor is able to assist in execution of SQL functions (both aggregate and scalar), SQL `LIMIT` operations, index scans, and table scans. All join operations, however, will be performed as root tasks.
```

**Comment by morgo** [comment]
@dcalvin PTAL again - I noticed grammar close to my edit was not good, so I improved it.

You will also need to sign the CLA :-)

**Pull request merged**